src/main/java/nextstep/subway/controller/FavoriteController.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.subway.controller;
+
+import lombok.RequiredArgsConstructor;
+import nextstep.configuration.auth.AuthenticationPrincipal;
+import nextstep.member.controller.dto.LoginMember;
+import nextstep.member.domain.entity.Member;
+import nextstep.member.domain.query.MemberReader;
+import nextstep.subway.controller.dto.CreateFavoriteRequest;
+import nextstep.subway.domain.command.FavoriteCommand;
+import nextstep.subway.domain.command.FavoriteCommander;
+import nextstep.subway.domain.query.FavoriteReader;
+import nextstep.subway.domain.view.FavoriteView;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping("/favorites")
+@RequiredArgsConstructor
+public class FavoriteController {
+    private final FavoriteCommander favoriteCommander;
+    private final FavoriteReader favoriteReader;
+    private final MemberReader memberReader;
+
+    @PostMapping("")
+    public ResponseEntity<FavoriteView.Main> createFavorite(
+            @AuthenticationPrincipal LoginMember loginMember,
+            @RequestBody CreateFavoriteRequest request
+    ) {
+        Member member = memberReader.getMe(loginMember.getEmail());
+        Long id = favoriteCommander.createFavorite(request.toCommand(member.getId()));
```

Comment:
```suggestion
    @PostMapping("")
    public ResponseEntity<FavoriteView.Main> createFavorite(
            @AuthenticationPrincipal LoginMember loginMember,
            @RequestBody CreateFavoriteRequest request
    ) {
        Long id = favoriteCommander.createFavorite(request.toCommand(loginMember));
```

각 즐겨찾기에 대한 api 마다 member를 찾는 로직이 들어가 있는데요. 이렇게 되면, 인증이 필요한 모든 api는 member를 찾도록 할 것 같아요. 어떻게 해야 member를 찾는 로직을 생략할 수 있을까요? 🤔

Reply:
Token 에 member 데이터를 넣는 것을 선호하는데 다음 스탭에 변경해보겠습니다!.